# 3D position of sonar point
float32 X
float32 Y
float32 Z
bool initDone
